### ISSUE TITLE
Admin settings missing after 3.3

### ISSUE THREAD

**Issue by asdraper27** [OfficeDev/microsoft-teams-emergency-operations-center#291]
I just recently performed the upgrade to 3.3, and the Teams admin center is showing the updated version.  I created the Teams app package and uploaded it.  When I go into Teams, I still see version 3.1, but now the Admin settings page is blank.  The map viewer is still mentioning setting it up via Bing Maps as well.  Do I need to uninstall the app and re-add it to Teams?

**Comment by v-ajaysahu** [comment]
@asdraper27 , thank you for notifying us. 
Could you confirm if you've updated the latest changes in the Azure App Service? If so, please check and let us know if the update was successful. Please follow below link for more information.
https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/wiki/Upgrade-to-Latest-Version#6-sync-latest-version

**Comment by asdraper27** [comment]
Yes, I synced the App Service to the new version first, then created the app package and uploaded the new file to Teams Admin Center.  
![Screenshot 2024-11-18 162732](https://github.com/user-attachments/assets/d57cef00-7367-465c-8cd0-c375892627b7)

When I go to Teams to open the app, I can see the overview which states it is on Version 3.3:

![image](https://github.com/user-attachments/assets/82a385e8-7564-4347-9e3a-4feb6b9b3b1c)


But when I open the app itself, and go to Info, it shows here as 3.1:  

![image](https://github.com/user-attachments/assets/629ea752-6eea-47f8-9535-953d37dd91a2)

The Admin setting menu option just presents me with a blank page in Teams, and the Map Viewer still has the message about needing to set up Bing Maps.

**Comment by lawrence-jeff** [comment]
I am seeing the same thing. I notice in the web client the info (of the site not the Teams app) shows 3.3, but the teams fat app continues to show v3.2 -perhaps some sort of caching issue?

**Comment by v-saikirang** [comment]
Hi @asdraper27,

This could be a caching issue. Please clear your cache and try again. Also check Teams in incognito browser to make sure.
Thanks.

**Comment by asdraper27** [comment]
Excellent, that worked for the browser-based app, but how do I clear cache for the desktop application?  And I have an Azure maps subscription - I pasted in the key and it said it was enabled, but I now have a blank map screen with my location pinned on it:

![image](https://github.com/user-attachments/assets/49a2ac9a-c2cd-4d34-b339-c538d493dc64)

**Comment by v-saikirang** [comment]
Hi @asdraper27,

Please check this for clearing cache: https://learn.microsoft.com/en-us/microsoftteams/troubleshoot/teams-administration/clear-teams-cache

Can you please share screenshot of the Config Settings page and also share console logs from browser after opening the maps tab on the app.
Thanks.

**Comment by asdraper27** [comment]
I did the reset of the Teams app, and the updated version is now showing, thank you.  I don't love that our users have to blow away all their Teams settings to get the update, but we will certainly survive.

I had the Azure Maps Subscription key added, but it initially did not work.  I had to regenerate the key, and now the Maps are connected correctly.  Are you aware if there is a life span for that key, will we need to regenerate it on a schedule?
